src/Service/CloudWatchLogs/tests/Unit/Input/DescribeLogStreamsRequestTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Tests\Unit\Input;
+
+use AsyncAws\CloudWatchLogs\Input\DescribeLogStreamsRequest;
+use AsyncAws\Core\Test\TestCase;
+
+class DescribeLogStreamsRequestTest extends TestCase
+{
+    public function testRequest(): void
+    {
+        self::fail('Not implemented');
+
+        $input = new DescribeLogStreamsRequest([
+            'logGroupName' => 'change me',
+            'logStreamNamePrefix' => 'change me',
+            'orderBy' => 'change me',
+            'descending' => false,
+            'nextToken' => 'change me',
+            'limit' => 1337,
+        ]);
+
+        // see https://docs.aws.amazon.com/cloudwatchlogs/latest/APIReference/API_DescribeLogStreams.html
```

Comment:
Sometimes, the generator fails to guess the url.
The Correct url is https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html

(same for all others tests)